packages/react-composites/src/composites/common/PeoplePaneContent.tsx

```diff
@@ -42,17 +44,21 @@ export const PeoplePaneContent = (props: {
   alternateCallerId?: string;
 }): JSX.Element => {
   const { inviteLink, onFetchParticipantMenuItems, setDrawerMenuItems, strings, onRemoveParticipant } = props;
-
+  /* @conditional-compile-remove(rooms) */
```

Comment:
Add a `hasRemoveParticipantsPermissionTrampoline` helper function that returns `false` on stable build, and remove all conditional compilation from this component.

Reply:
done